ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -921,13 +921,17 @@ loopback_interfaces:
     mpls:
       ldp:
         interface: < true | false >
+
   < Loopback_interface_2 >:
```

Comment:
Will it be confusing to user if ```< Loopback_interface_1 >``` and ```< Loopback_interface_2 >``` have different set of parameters? Please comment!

Reply:
This was already present before this PR so i just added a few lines to it. I think we can keep it as it is.

Reply:
ok